common/models/model-definition.json

```diff
@@ -0,0 +1,53 @@
+{
+  "properties": {
+    "id": {
+      "type": "string",
+      "id": true,
+      "json": false
+    },
+    "facetName": {
```

Comment:
No, not yet, but this is required very soon to create rest models for model definitons